custom_components/petkit/translations/bg.json

```diff
@@ -0,0 +1,743 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "PetKit акаунтът вече е конфигуриран",
+      "reauth_successful": "Повторната автентификация беше успешна"
+    },
+    "error": {
+      "cannot_connect": "Неуспешно свързване",
+      "invalid_auth": "Невалидна автентификация. Правилни ли са вашите данни за вход?",
+      "no_devices": "Не са открити устройства на акаунта",
+      "server_busy": "Сървърите на PetKit са заети. Моля, опитайте отново по-късно.",
+      "petkit_error": "Възникна неизвестна грешка. Моля, вижте логовете на Home Assistant за повече подробности.",
+      "region_error": "Моля, изберете държавата, свързана с вашия акаунт.",
+      "timezone_error": "Часовата зона не може да бъде намерена. Ако използвате Home Assistant като самостоятелен Docker контейнер, трябва да дефинирате променливата TZ."
```

Comment:
Please look at the current en.json file, the timezone_error was updated before the last release.